barretenberg/cpp/src/barretenberg/proof_system/op_queue/ecc_op_queue.hpp

```diff
@@ -7,6 +7,17 @@ namespace bb {
 
 enum EccOpCode { NULL_OP, ADD_ACCUM, MUL_ACCUM, EQUALITY };
 
+struct UltraOpTuple {
```

Comment:
It's slightly confusing, because it is not a tuple

Reply:
I guess we have ECCVMOp and this should be just ECCOp?

Reply:
I guess you previous suggestion for just naming it "UltraOp" is better